CMake/External_SWIG.cmake

```diff
@@ -0,0 +1,43 @@
+
+if(WIN32)
```

Comment:
Isn't python required for SWIG? You have a if(NOT fletch_BUILD_WITH_PYTHON) block in the tensorflow CMake file, but not here.